{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-23458",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "total"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-08-06T18:35:13.595878Z"
              }
            }
          }
        ],
        "affected": [
          {
            "cpes": [
              "cpe:2.3:a:zscaler:client_connector:*:*:*:*:*:windows:*:*"
            ],
            "vendor": "zscaler",
            "product": "client_connector",
            "versions": [
              {
                "status": "affected",
                "version": "0",
                "lessThan": "4.2.0.190",
                "versionType": "custom"
              }
            ],
            "defaultStatus": "unknown"
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-08-06T18:36:17.748Z"
        }
      }
    ],
    "cna": {
      "title": "Local Privilege Escalation on Zscaler Client Connector on Windows",
      "source": {
        "discovery": "UNKNOWN"
      },
      "credits": [
        {
          "lang": "en",
          "type": "finder",
          "value": "Equinor Red Team"
        }
      ],
      "impacts": [
        {
          "capecId": "CAPEC-233",
          "descriptions": [
            {
              "lang": "en",
              "value": "CAPEC-233 Privilege Escalation"
            }
          ]
        }
      ],
      "metrics": [
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 7.3,
            "attackVector": "LOCAL",
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:L",
            "integrityImpact": "HIGH",
            "userInteraction": "NONE",
            "attackComplexity": "LOW",
            "availabilityImpact": "LOW",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "HIGH"
          },
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "affected": [
        {
          "vendor": "Zscaler",
          "product": "Client Connector",
          "versions": [
            {
              "status": "affected",
              "version": "0",
              "lessThan": "4.2.0.190",
              "versionType": "custom"
            }
          ],
          "platforms": [
            "Windows"
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "references": [
        {
          "url": "https://help.zscaler.com/client-connector/client-connector-app-release-summary-2023?applicable_category=windows&applicable_version=4.2.0.190"
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "While copying individual autoupdater log files, reparse point check was missing which could result into crafted attacks, potentially leading to a local privilege escalation. This issue affects Zscaler Client Connector on Windows <4.2.0.190.",
          "supportingMedia": [
            {
              "type": "text/html",
              "value": "While copying individual autoupdater log files, reparse point check was missing which could result into crafted attacks, potentially leading to a local privilege escalation. This issue affects Zscaler Client Connector on Windows &lt;4.2.0.190.<br>",
              "base64": false
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-346",
              "description": "CWE-346 Origin Validation Error"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "73c6f63b-efac-410d-a0a9-569700f85a04",
        "shortName": "Zscaler",
        "dateUpdated": "2024-08-06T15:22:58.963Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2024-23458",
    "state": "PUBLISHED",
    "dateUpdated": "2024-08-06T18:36:26.478Z",
    "dateReserved": "2024-01-17T15:15:47.221Z",
    "assignerOrgId": "73c6f63b-efac-410d-a0a9-569700f85a04",
    "datePublished": "2024-08-06T15:22:58.963Z",
    "assignerShortName": "Zscaler"
  },
  "dataVersion": "5.1"
}